routes/tasks.js

```diff
@@ -159,7 +162,7 @@ router.post("/", authenticate, authorizeUser("appOwner"), createTask, tasks.addN
  *           schema:
  *             $ref: '#/components/schemas/errors/badImplementation'
  */
-router.patch("/:id", authenticate, authorizeUser("appOwner"), updateTask, tasks.updateTask);
+router.patch("/:id", authenticate, authorizeRoles([APPOWNER]), updateTask, tasks.updateTask);
```

Comment:
Previous middleware `authorizeUser("appOwner")` gives permission to both app_owener and super_user access. 
The current only gives it to mentioned role. So can we add SUPERUSER to the array as well? 
```suggestion
router.patch("/:id", authenticate, authorizeRoles([APPOWNER, SUPERUSER]), updateTask, tasks.updateTask);
```

Reply:
okh that means where ever APPOWNER role is given, we need to pass SUPERUSER role also in that array, am i understanding it right? @pallabez